src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -1010,6 +1012,20 @@ static <R> R extractUniqueDrsObjectValue(
     }
   }
 
+  /** Given a list of DRSObjects, concatenate the string values of the mapped field */
+  @VisibleForTesting
+  static <R> String extractDrsFileConcatenatedStringValues(
+      List<DRSObject> drsObjects, Function<DRSObject, ? extends R> mapper) {
+    return drsObjects.stream()
+        .map(mapper)
+        .filter(Objects::nonNull)
+        .map(Objects::toString)
+        .map(String::trim)
+        .distinct()
+        .filter(Predicate.not(String::isEmpty))
+        .collect(Collectors.joining(", "));
+  }
```

Comment:
This is fine although the current use case is a method that returns `String`. So you could simplify a little by changing It to use `String` instead of generic type `R`.
```suggestion
  static String extractDrsFileConcatenatedStringValues(
      List<DRSObject> drsObjects, Function<DRSObject, String> mapper) {
    return drsObjects.stream()
        .map(mapper)
        .filter(Objects::nonNull)
        .filter(Predicate.not(String::isEmpty))
        .distinct()
        .collect(Collectors.joining(", "));
  }
```

Also I wouldn't bother with `trim()` here.